### PR TITLE
Merge po

### PR DESCRIPTION
**Pull request by doniks**

### CODE REVIEW

**Comment by doniks** [comment]
@peat-psuwit @Danfro ready for review

to demonstrate that the structure should be ok:
* here a dummy rtd setup:
  * EN: https://docs.ubports.com/en/merge_po
  * DE: https://docs.ubports.com/de/merge_po
* and a dummy weblate setup:
  * https://hosted.weblate.org/projects/ubports/docs-ubports-com/#translations

**Comment by doniks** [comment]
btw. I'd prefer the commits *not* be squashed. they are not all 100% clean, but still explain the pot surgery better than if they were squashed

File: merge_po.sh [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
     lang_code=$(basename "$locale_dir")
     d=$locale_dir/LC_MESSAGES
 
+    # for s in $sections ; do
+    #     ls -l $d/$s.po
+    #     # --no-fuzzy-matching
+    #     msguniq --width=76 $d/$s.po -o $d/${s}_clean.po
+    # done
+
+    # msgcat --unique $d/index.po $d/about.po $d/appdev.po $d/contribute.po $d/humanguide.po $d/porting.po $d/systemdev.po $d/userguide.po -o $d/$lang_code.po
```
Comment (peat-psuwit) [inline merge_po.sh:19]: Please don't include commented code. It's confusing

File: merge_po.sh [hunk 2]
```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
 
+# set -e
+# set -xv
+
+sections="index about appdev contribute humanguide porting systemdev userguide"
```
Comment (peat-psuwit) [inline merge_po.sh:6]: Does this variable turn out to be used by anything else?

**Review by peat-psuwit — COMMENTED** [review]
I think they look good. But if you prefer the commits not to be squashed, then please also cleanup git commit messages. For example, "unbreakeroo" is not a good commit message. `git rebase -i` is a good tool to do that.

Actually, preferably the commits themselves should also be cleaned up. But I guess that could be more complicated.

**Comment by doniks** [comment]
okok. I'll clean it up

**Comment by doniks** [comment]
ok. now it comes as a clean commit history that accurately documents the steps performed for this pot surgery. @peat-psuwit please re-review

**Review by peat-psuwit — APPROVED** [review]

**Pull request merged**